### PR TITLE
Enable build gating on development branch

### PR DESCRIPTION
**Pull request by quic-mtharu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ https://developer.qualcomm.com/software/hexagon-dsp-sdk
 ###Steps to generate native binaries on device
 
 ```
-git clone https://github.com/quichub/fastrpc
+git clone https://github.com/quic/fastrpc
```
Comment (ekanshibu) [inline README.md]: let keep this as a separate PR

Comment (quic-mtharu) [reply README.md]: removed this commit changes for this PR

**Review by quic-ekangupt — APPROVED** [review]
LGTM
